linkis-engineconn-plugins/spark/src/main/scala/org/apache/linkis/engineplugin/spark/Interpreter/ProcessInterpreter.scala

```diff
@@ -33,8 +33,6 @@ import java.util.concurrent.TimeUnit
 import scala.concurrent.{Await, ExecutionContext, Future}
 import scala.concurrent.duration.Duration
 
-import org.json4s._
-
 /**
  */
 abstract class ProcessInterpreter(process: Process) extends Interpreter with Logging {
```

Comment:
can delete ProcessInterpreter，No longer use

Reply:
seems `PythonInterpreter` still extends from `ProcessInterpreter `

```
private class PythonInterpreter(process: Process, gatewayServer: GatewayServer)
    extends ProcessInterpreter(process)
    with Logging {
```

Reply:
@GuoPhilipse Yes, but it is no longer used, we can remove inheritance and remove unused methods